docs/content/installation/migration/ingress-nginx.md

```diff
@@ -0,0 +1,549 @@
+
+**Ingress-NGINX Controller**
+```yaml
+nginx.ingress.kubernetes.io/auth-tls-secret: secretName
+nginx.ingress.kubernetes.io/auth-tls-verify-client: "on"
+nginx.ingress.kubernetes.io/auth-tls-verify-depth: "1"
+```
+
+**NGINX Ingress Controller**
+```yaml
+ingressMTLS:
+   clientCertSecret: secretName
+   verifyClient: "on"
+
+   verifyDepth: 1
+```
+
+---
+
+**Ingress-NGINX Controller**
+```yaml
+nginx.ingress.kubernetes.io/proxy-ssl-secret: "secretName"
+nginx.ingress.kubernetes.io/proxy-ssl-verify: "on|off"
+nginx.ingress.kubernetes.io/proxy-ssl-verify-depth: "1"
+nginx.ingress.kubernetes.io/proxy-ssl-protocols: "TLSv1.2"
+nginx.ingress.kubernetes.io/proxy-ssl-ciphers: "DEFAULT"
+nginx.ingress.kubernetes.io/proxy-ssl-name: "server-name"
+nginx.ingress.kubernetes.io/proxy-ssl-server-name: "on|off"
+```
+
+**NGINX Ingress Controller**
+```yaml
+egressMTLS:
+   tlsSecret: secretName
+
+   verifyServer: true|false
+
+   verifyDepth: 1
+
+   protocols: TLSv1.2
+
+   ciphers: DEFAULT
+
+   sslName: server-name
+
+   serverName: true|false
+```
+
+---
+
+#### Session Persistence with NGINX Plus
+With [NGINX Plus]({{<relref "overview/nginx-plus">}}), you can use [Policy]({{<relref "configuration/policy-resource">}}) resources for session persistence, which have corresponding annotations for the community Ingress-NGINX Controller.
+
+**Ingress-NGINX Controller**
+```yaml
+nginx.ingress.kubernetes.io/affinity: "cookie"
+nginx.ingress.kubernetes.io/session-cookie-name: "cookieName"
+nginx.ingress.kubernetes.io/session-cookie-expires: "x"
+nginx.ingress.kubernetes.io/session-cookie-path: "/route"
+nginx.ingress.kubernetes.io/session-cookie-secure: "true"
+```
+
+**NGINX Ingress Controller**
+```yaml
+sessionCookie:
+  enable: true
+
+  name: cookieName
+
+  expires: xh
+
+  path: /route
+
+  secure: true
+```
+
+## Migration with Kubernetes Ingress Resources
+The other option for migrating from the community Ingress-NGINX Controller to NGINX Ingress Controller is using only [annotations](https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/) and [ConfigMaps](https://kubernetes.io/docs/concepts/configuration/configmap/) from standard Kubernetes resources, potentially relying on [mergeable Ingress types](https://github.com/nginxinc/kubernetes-ingress/tree/main/examples/ingress-resources/mergeable-ingress-types).
+
+This ensures that all configuration is kept in the Ingress object.
+
+{{< warning >}}
+Do not alter the `spec` field of the Ingress resource when taking this option.
```

Comment:
Why? It might be helpful for a warning to explain the consequences -- could help w/ troubleshooting.

Reply:
I have no idea why, but I'll ask Jason Williams later - I suspect changing the spec fields breaks the resource's ability to parse other parameters.

Reply:
A warning is the highest level of caution, intended for issues that could lead to data loss or hardware failure. We need to specify the risks to keep users informed. If the consequences are unclear, perhaps this shouldn't be labeled as a warning.